tasks/main.yml

```diff
@@ -6,10 +6,20 @@
 # * Install, Activate and Manage Trend Micro Deep Security Agent
 # *********************************************************************
 ---
+- name: "Announce deprecated for variable 'Action'"
+  debug:
+    msg: "[Warning] the variable 'action' is deprecated, please change to use 'operation instead'"
+  when: action is defined and (operation is not defined)
```

Comment:
I think we should only check if action is used, even if operation is used, the customer should not use action anymore.

Reply:
Check the position of quote mark. It should be "use 'operation' instead"

Reply:
@rogeryang777 Sure, will change in next commit.
@chchhsiao i think the point is to urge users to use "operation" instead of "action", so i'll keep what it is here.